Gemfile

```diff
@@ -47,6 +47,9 @@ gem 'puma', '~> 2.16'
 # Used to colorize output for rake tasks
 gem "rainbow"
 
+# Carmen for state names
+gem 'carmen-rails', '~> 1.0', '>= 1.0.1'
```

Comment:
`carmen-rails` is no longer maintained. Are there alternatives?

https://github.com/jim/carmen-rails